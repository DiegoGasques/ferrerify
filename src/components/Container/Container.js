import React from "react";
import "./Container.scss";

export const Container = ({ children }) => {
  return <div className="Container">{children}</div>;
};

export default Container;
