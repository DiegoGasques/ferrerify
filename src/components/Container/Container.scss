.Container {
  display: flex;
  max-width: 950px;
  margin: 0 auto;
}
