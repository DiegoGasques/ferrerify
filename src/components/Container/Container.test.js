import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { Container } from "./Container";

describe("[COMPONENT] Container", () => {
  it("Tests for changes in the rendered component", () => {
    const wrapper = shallow(<Container />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
