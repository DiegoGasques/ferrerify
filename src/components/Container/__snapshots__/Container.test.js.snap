// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] Container Tests for changes in the rendered component 1`] = `
<div
  className="Container undefined"
/>
`;
